gtsfm/utils/metrics.py

```diff
@@ -120,20 +122,19 @@ def compute_translation_angle_metric(
         wTi_list: List of estimated camera poses.
 
     Returns:
-        A statistics dict of the metrics errors in degrees.
+        A GtsfmMetric for the translation angle errors, in degrees.
     """
     angles = []
     for (i1, i2) in i2Ui1_dict:
         i2Ui1 = i2Ui1_dict[(i1, i2)]
         angles.append(comp_utils.compute_translation_to_direction_angle(i2Ui1, wTi_list[i2], wTi_list[i1]))
-    return GtsfmMetric("translation_angle_deg", np.array(angles, dtype=np.float))
+    return GtsfmMetric("translation_angle_error_deg", np.array(angles, dtype=np.float))
 
 
 def compute_averaging_metrics(
-    i2Ui1_dict: Dict[Tuple[int, int], Unit3],
     wRi_list: List[Optional[Rot3]],
     wti_list: List[Optional[Point3]],
-    gt_wTi_list: List[Optional[Pose3]],
+    gt_wTi_list: List[Pose3],
```

Comment:
well we technically might have some GT poses available and some not, or maybe I am thinking too much.

Reply:
@ayushbaid good point, I considered that. However my current thinking is that we should only use datasets that have GT poses for all frames, or no frames. Otherwise things will get too confusing.

Reply:
Sure.